datagouv-components/src/components/ResourceAccordion/Pmtiles.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div
+      v-if="hasError"
+      class="bg-warning-lightest text-warning-dark p-3 mt-8 mx-8 mb-3"
+    >
+      <p class="fr-grid-row fr-m-0">
+        <span
+          class="fr-icon-warning-line"
+          aria-hidden="true"
+        />
+        {{ t("The map preview of this file failed to load.") }}
+      </p>
+    </div>
+    <template v-else>
+      <div class="bg-blue-100 text-datagouv fr-hidden fr-unhidden-md p-4">
+        <div class="fr-grid-row fr-grid-row--middle fr-grid-row--gutters">
+          <div
+            class="fr-col-auto"
+            v-html="franceSvg"
+          />
+          <div class="fr-col">
+            <p class="fr-text--bold fr-m-0">
+              {{ t("Explore data in detail") }}
+            </p>
+            <p class="fr-text--sm fr-m-0 f-italic">
+              {{ t("Use PMTiles viewer to get an overview of geojson data.") }}
+            </p>
+          </div>
+          <p class="fr-col-auto fr-my-0">
+            <BrandedButton
+              :href="pmtilesViewerUrl"
+              :icon="RiExternalLinkFill"
+              icon-right
+            >
+              {{ t("Explore map") }}
+            </BrandedButton>
+          </p>
+        </div>
+      </div>
+      <div style="height: 600px;" ref="containerRef" />
+      <div class="fr-px-5v fr-pt-5v">
+        {{ t("Map preview updated on {date}", { date: lastUpdate }) }}
+      </div>
+    </template>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, onMounted, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { RiExternalLinkFill } from '@remixicon/vue'
+import { Protocol, PMTiles } from 'pmtiles'
+import maplibregl from 'maplibre-gl'
+import DOMPurify from 'dompurify'
+import { formatDate } from '../../functions/dates'
+import type { Resource } from '../../types/resources'
+import BrandedButton from '../BrandedButton.vue'
+import styleVector from '../../../assets/json/vector.json'
+import franceSvg from './france.svg?raw'
+
+const props = defineProps<{ resource: Resource }>()
+
+const { t } = useI18n()
+
+const hasError = ref(false)
+const pmtilesUrl = props.resource.extras['analysis:parsing:pmtiles_url']
```

Comment:
It's better to have computed (even if in this case the `props.resource` will not change at runtime, we never know…)

Reply:
Done in https://github.com/datagouv/cdata/pull/438/commits/e516dfee915b266ddfec6f418ef1a17272ca36d1